railties/lib/rails/generators/rails/sessions/templates/controllers/concerns/authentication.rb

```diff
@@ -37,7 +37,7 @@ def find_session_by_cookie
 
     def request_authentication
       session[:return_to_after_authenticating] = request.url
-      redirect_to new_session_url
+      redirect_to new_session_url, alert: "Please log in to continue."
```

Comment:
Maybe a bit too much with the alert here. When app redirects to sign in screen it is clear what happened without the **alert**

Reply:
I think it's a good pattern to add flash messages to redirects like "require authentication/require authentication". Anyway, it's quite out of scope of what I initially wanted to fix with this PR -> removed 🙌